packages/core/src/utils/schema.ts

```diff
@@ -18,3 +18,12 @@ export const ChainSchema = z.custom<Chain>((chain) => {
     return false;
   }
 });
+
+export const PercentageSchema = z.object({
+  /**
+   * Percent value between 1 and 1000 inclusive for maxPriorityFeePerGas estimate added buffer.
+   * maxPriorityFeePerGasBid is set to the max between the buffer "added" priority fee estimate and
+   * the minPriorityFeePerBid (default: 33)
+   */
+  percentage: z.number().min(1).max(1000).default(33),
```

Comment:
Does this require a default? Is this required for backwards compatibility? This schema is used for more fields than the fees now and a 33% default doesn't make sense for each value.

Reply:
yea agree with @dancoombs here, let's not add a default on the percentage schema. instead places that use the schema can specify the default themselves

also same question as earlier (i may have missed your comment), but why is the `max(1000)`? i imagine it should either be `100` or unbounded

Reply:
@moldy530 per https://www.notion.so/alchemotion/2023-08-10-Alchemy-Gas-and-Paymaster-Update-020b41bee84641db98210cb71bca96af?pvs=4#b791899b4f7242c998cf5522fa734bb3

Reply:
synced offline with @moldy530 . To avoid not making the aa-sdk providers code not too messy with overused generics, will leave it as introducing these percentage types at the base level, at least for now. cc: @dancoombs @moldy530